tests/morpheus/pipeline/test_multi_port_pipeline.py

```diff
@@ -14,13 +14,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# When segment modules are imported, they're added to the module registry.
+# To avoid flake8 warnings about unused code, the noqa flag is used during import.
+import modules.multiplexer  # noqa: F401 # pylint: disable=unused-import
```

Comment:
tests/morpheus/modules is somewhat overloaded here. It is used for -
1. unit-tests associated with python/morpheus/modules (AND)
2. for common test functions like modules.multiplexer which are imported without install.

David, Do you think we should split the dir up to avoid confusion?